src/Models/HydrostaticFreeSurfaceModels/implicit_free_surface.jl

```diff
@@ -25,7 +25,11 @@ ImplicitFreeSurface(; gravitational_acceleration=g_Earth) =
 
 # Internal function for HydrostaticFreeSurfaceModel
 function FreeSurface(free_surface::ImplicitFreeSurface{Nothing}, velocities, arch, grid)
-    η = CenterField(arch, grid, TracerBoundaryConditions(grid))
+    bcs=TracerBoundaryConditions(grid,
+       west=BoundaryCondition(Gradient, Float64(0)), east=BoundaryCondition(Gradient, Float64(0)),
+       north=BoundaryCondition(Gradient, Float64(0)), south=BoundaryCondition(Gradient, Float64(0)),
```

Comment:
I think maybe you want to use `BoundaryCondition(Flux, nothing)` here? This is our notation for a "zero flux" boundary condition.